rtlib/CMakeLists.txt

```diff
@@ -40,6 +40,14 @@ target_link_libraries(${target}
         ITKCommon
         ITKTransform
         ITKOptimizers
+	vtkCommonCore
+	vtkCommonDataModel
+	vtkCommonExecutionModel
+	vtkCommonMath
+	vtkCommonMisc
+	vtkCommonTransforms
+	vtkFiltersCore
+	vtkFiltersGeneral
```

Comment:
If things are included in headers, they should be added to PUBLIC linkage. If in a cpp, to PRIVATE linkage. This looks like the minimal set of VTK libraries for `rtlib`:
```cmake
PUBLIC
    vtkCommonCore
    vtkCommonDataModel
PRIVATE
    vtkCommonTransforms
    vtkFiltersGeneral
```
You should be able to `make rtlib` on its own to test this. If, after that, any of the apps fail to build, they likely are adding link dependencies not required by `rtlib`.